mesonbuild/modules/features/feature.py

```diff
@@ -0,0 +1,356 @@
+# Copyright (c) 2023, NumPy Developers.
+# All rights reserved.
```

Comment:
We've moved to using SPDX identifiers for the license, so we should have that there.

Also, this `All rights reserved` seems spurious, since to land in Meson it will need to be under the Apache-2.0 or a compatible permissive license like the MIT or BSD-2-clause